lib/preprocessCss.js

```diff
@@ -223,7 +223,7 @@ module.exports = (app, callback) => {
       }
 
       // compare existing content to new content before generating
-      if (!versionData && versionData !== versionCode) {
+      if (!versionData || versionData !== versionCode) {
```

Comment:
@Autre31415 does this looks okay, I didn't really play around with the version file at all

Reply:
Yeah this looks right. It matches the rest of it now